lib/holdings/mhld.rb

```diff
@@ -48,7 +48,7 @@ def standard_or_zombie_library
     end
 
     def sanitize_mhld_data(data)
-      CGI.escape_html(data).gsub('),', '), ').gsub('-', '-<wbr/>').html_safe if data.present?
+      CGI.escape_html(data).gsub('),', '), ').gsub(/,\s?/, ', ').gsub('-', '-<wbr/>').html_safe if data.present?
```

Comment:
I'm late to chime in, but couldn't we add a `<wbr />` after commas too and keep the two displays similar?

Reply:
Yeah that would also work fine too. Though I think we still have to do the thing for librarian view as we aren't sanitizing anything there that is just the value.

@jvine do you have a pref for space vs `<mbr/>` ?

Reply:
Going with what we have for now.